pkgs/tools/misc/mons/default.nix

```diff
@@ -33,5 +72,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/Ventto/mons.git";
     license = licenses.mit;
     maintainers = with maintainers; [ mschneider thiagokokada ];
+    platforms = platforms.unix;
```

Comment:
Technically it is Linux only for some features (like the daemon), but it should work in any Unix platform where X.org+Xrandr is available.